fe/fe-core/src/main/java/com/starrocks/server/StorageVolumeMgr.java

```diff
@@ -228,13 +224,19 @@ public void unbindTableToStorageVolume(long tableId) {
         }
     }
 
+    public String getStorageVolumeIdOfTable(long tableId) {
+        try (LockCloseable lock = new LockCloseable(rwLock.readLock())) {
+            return tableToStorageVolume.get(tableId);
+        }
+    }
+
     public String getStorageVolumeIdOfDb(long dbId) {
         try (LockCloseable lock = new LockCloseable(rwLock.readLock())) {
             return dbToStorageVolume.get(dbId);
         }
     }
 
-    public StorageVolume getDefaultStorageVolume() throws AnalysisException {
```

Comment:
AnalysisException can also be removed in `createStorageVolume`, please also help me remove it.